app/views/layouts/user_mailer.html.erb

```diff
@@ -59,10 +59,10 @@
                       <table class="row">
                         <tbody>
                           <tr>
-                            <th class="small-12 large-12 columns first last">
+                            <th class="small-12 large-12 columns first last" scope="row">
```

Comment:
Is the `scope` still necessary after we apply `role="presentation"`? I would expect `role="presentation"` to remove any semantic meaning of the table such that the `<th>` wouldn't represent a semantic heading requiring a `scope` attribute.

Reply:
It seems that `role="presentation"` in the containing table is sufficient to mark the embedded tables as presentation as well, so it does not appear to be needed.

Reply:
i thought it would be enough but when testing it out with the ibm accessibility checker there are some instances where  scope is required.

Reply:
**[Mozilla says](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/presentation_role)**
> If presentation or none is applied to a `<table>` element, the descendant `<caption>`, `<thead>`, `<tbody>`, `<tfoot>`, `<tr>`, `<th>`, and `<td>` elements inherit the role and are thus not exposed to assistive technologies. But, elements inside of the `<th>` and `<td>` elements, including nested tables, are exposed to assistive technologies.

Which makes me think that we could remove the `scope` from the `<th>` tags, but add `role="presentation"` to all the nested tables. That's 13 tables on this page. I'm trying this out locally, but the change seems to crash the IBM accessibility checker.

Reply:
i made an update that removes scope and uses the <td> element in replace of <th> and adds a presentation role to inner tables. While this clears the checks in the IBM checker it does not clear all of them in SonarQube (35 errors remain). Scope however cleared both IBM and SonarQube scans and requires fewer code additions, for that reason I think it makes sense to go back to using scope. I'll push up another change.